data/json/field_type.json

```diff
@@ -1993,5 +1993,453 @@
     "priority": 1,
     "display_field": true,
     "looks_like": "fd_shadow"
+  },
```

Comment:
Might want to move this to a new file so people can more easily add new zones.

Reply:
Good suggestion. Doing that in (in my next commit).